fdbclient/ClientWorkerInterface.h

```diff
@@ -88,4 +90,23 @@ struct ProfilerRequest {
 	}
 };
 
+struct SetFailureInjection {
+	constexpr static FileIdentifier file_identifier = 15439864;
+	ReplyPromise<Void> reply;
+	struct ThrottleDiskCommand {
+		double time;
+		Optional<NetworkAddress> address; // TODO: NEELAM: how do we identify the machine
```

Comment:
can you please document what `time` means here? Is this for how long throttling should be active or how much you throttle?